Sessions/3.2 - Reading XML Files.ipynb

```diff
@@ -770,89 +1256,158 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 17,
    "id": "742bdf0a",
    "metadata": {
     "slideshow": {
      "slide_type": "slide"
     }
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'stanza'"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "# each element has different __Python__ attributes\n",
-    "root[1][1].tag"
+    "root[1][2].tag"
```

Comment:
I got an `IndexError: list index out of range` error.

Reply:
I removed `root[1][2].tag`